src/modules/initcpiocfg/main.py

```diff
@@ -173,20 +181,28 @@ def modify_mkinitcpio_conf(partitions, root_mount_point):
     else:
         hooks.extend(["filesystems"])
 
-    if btrfs == "yes" and cpu['proc0']['vendor_id'].lower() != "genuineintel":
+    def is_cpu_intel():
+        cpu_intel = "genuineintel"
+        if cpu["proc0"]["vendor_id"].lower() == cpu_intel:
+            return True
+        else:
+            return False
```

Comment:
`return cpu["proc0"]["vendor_id"].lower() == "genuineintel"`

Or just make a variable out of it -- this doesn't have to be a function.

Reply:
I think this vendor check is not needed, it loads on AMD too.

```
crazy@crazy-nitroan51543:~/kernel/linux$ sudo modprobe btrfs
crazy@crazy-nitroan51543:~/kernel/linux$ dmesg | tail -n1 
[185152.927724] Btrfs loaded, crc32c=crc32c-intel
crazy@crazy-nitroan51543:~/kernel/linux$ uname -a
Linux crazy-nitroan51543 5.9.12-fw1 #1 SMP PREEMPT Wed Dec 2 17:32:29 CET 2020 x86_64 AMD Ryzen 7 3750H with Radeon Vega Mobile Gfx AuthenticAMD GNU/Linux
```